noxfile.py

```diff
@@ -52,10 +52,19 @@ def install(session: nox.Session, group: str) -> None:
         session.run(*"git clone https://github.com/laminlabs/lamin-cli".split())
         session.run(*"pip install lamin-cli".split())
     elif group == "hub-local":
-        session.run(*"pip install -e .[aws,dev,hub]".split())
-        session.run(*"pip install ./laminhub/rest-hub".split())
+        session.run(*"pip install lamindb".split())
```

Comment:
Ouch, this will get us into trouble, pulling a pypi lamindb version; we shouldn't do this, I'm pretty sure.